dask_saturn/core.py

```diff
@@ -16,16 +17,25 @@
 
 
 class SaturnCluster(SpecCluster):
-    def __init__(self, cluster_url=None, reset=False, *args, **kwargs):
+    def __init__(self, cluster_url=None, *args, **kwargs):
         if cluster_url is None:
-            self._start(reset=reset, **kwargs)
+            self._start(**kwargs)
         else:
             self.cluster_url = cluster_url if cluster_url.endswith("/") else cluster_url + "/"
         info = self._get_info()
         self._dashboard_link = info["dashboard_link"]
         self._scheduler_address = info["scheduler_address"]
         self.loop = None
         self.periodic_callbacks = {}
+    
+    @classmethod
+    def reset(cls, **kwargs):
```

Comment:
This needs a docstring for sure :)

Reply:
Also the kwargs should be listed explicitly.

Reply:
I've been going back and forth on listing all the kwargs. Didn't do it originally on init because there are so many of them, but since this is customer facing I can see why it would be useful.

Reply:
should we remove the args and kwargs entirely from init? Neither is actually being used for anything (or they won't be once I list the kwargs explicitly)

Reply:
I'm in favor of explicit arguments. Using `**kwargs` doesn't make the issue of "there are so many" go away...it just hides that fact from users, in my opinion

Reply:
Well really what I meant by "there are so many of them" is that it's a long list of things that we need to sync between here and the API. By using kwargs, we could easily add new configurations without needing to change this code.

But I do totally understand why being explicit is a good idea.